eppo_core/src/lib.rs

```diff
@@ -80,4 +80,5 @@ pub use attributes::{
 };
 pub use configuration::Configuration;
 pub use error::{Error, EvaluationError, Result};
+pub use sdk_key::SdkKey;
```

Comment:
This technically needs a minor version bump unless we feature-gate it under `event_ingestion`

Reply:
we probably should gate it with others since this should only be used in the `event_ingestion` codepath